sacred/observers/mongo.py

```diff
@@ -8,14 +8,15 @@
 import sys
 import time
 
+import pymongo
```

Comment:
Please put this import into the find_queued function. The reason is that `mongo.py` should be importable even if `pymongo` is not installed.